roles/libvirt_manager/tasks/clean_layout.yml

```diff
@@ -2,10 +2,15 @@
 - name: Get installed packages list
   ansible.builtin.package_facts: {}
 
-- name: Get virtqemud socket
-  register: _virtqemud
-  ansible.builtin.stat:
-    path: "/var/run/libvirt/virtqemud-sock"
+- name: Populate service facts
+  ansible.builtin.service_facts:
+
+- name: Start virtqemud socket service
+  ansible.builtin.service:
+    name: virtqemud.socket
+    state: started
+    enabled: yes
+  when: ansible_facts['services']['virtqemud.socket']['status'] | default('not-found') != 'not-found'
```

Comment:
wondering if that condition would be not better:
```
"'virtqemud.socket' not in ansible_facts.services"
```

Reply:
I would say that no because the ansible module doesn't list the socket service only the main service and we would like to make sure the service exist before trying to start the socket and interact with it.

Reply:
generally I agree but I feel this also solves the issue of it not being started